web/app/controllers/stateRegsCtrl.js

```diff
@@ -0,0 +1,16 @@
+var StateRegulations = require("../models/StateRegulations");
+
+exports.getStateRegs = function (state, callback, error) {
+	StateRegulations.findOne({
+		state: state
```

Comment:
can we add a check for string casing and whitespace? any extra whitespace and upper/lowercase might cause this to fail a query